inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/ctc_greedy_decoder_seq_len.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include "single_layer_tests/ctc_greedy_decoder_seq_len.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace ngraph::helpers;
+
+namespace {
+
+std::vector<std::vector<size_t>> inputShape{{1, 1, 1}, {1, 6, 10}, {3, 3, 16}, {5, 3, 55}};
+
+const std::vector<InferenceEngine::Precision> probPrecisions = {
+    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::FP16
+};
+const std::vector<InferenceEngine::Precision> idxPrecisions = {
+    InferenceEngine::Precision::I32,
+    //TODO: currently there is a bug in transformation.
+//    InferenceEngine::Precision::I64
```

Comment:
In general it is better to use skip_config for such purposes (also including issue number in the comment).

Reply:
I've added the ticket number.
To use DISABLE_ prefix need to divide instantiation by precision. It's inconvenient.

Reply:
I mentioned this skip config: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp. This is better to contain all disabled cases in one file. That helps us to avoid searching throw all the tests each time.

Reply:
Fixed